templates/partials/cancel-and-return.html

```diff
@@ -0,0 +1,10 @@
+{% block cancel_and_return %}
+  {% if previous_location_url %}
+    <p class="u-mt-l">
+      {% if content.block.cancel_text %}
+        {{ content.block.cancel_text }}
+      {% endif %}
+      <a id="cancel-and-return" href="{{ previous_location_url }}">{{ _("Return to the previous page") }}</a>
+    </p>
+  {% endif %}
+{% endblock cancel_and_return %}
```

Comment:
We should rename this file `list-action.tmpl`, extend from `question.html` and amend the add/edit/remove templates to inherit from `list-action.tmpl`. That way it's clear that add/edit/remove all use the same functionality.